Feature/sandbox nav item

emclaug2 opened the pull request:
<!-- Include a bulleted list summarizing the main changes you have made in this PR -->
#### Changes proposed in this Pull Request:
- Add Sandbox for Nav Item

emclaug2 commented:
@huayunh This PR should be ready for design eyes.  I've made the sandbox full-screen & fixed the overflow issue on the other pages by hiding the drawer at 1280px.

huayunh commented:
Looks lovely! I like those hint texts.

- I would add a `selected` knob and enable it by default.
- I would also do the hierarchy this way:

![image](https://user-images.githubusercontent.com/8997218/182628919-53713f93-358a-4320-8da8-faef70f87597.png)

Other than that, there may be some style updates in the future, but for the first pass this looks good.

emclaug2 commented:
Ok @huayunh I think this is the final version.

- I've added the `selected` knob.
- Adjusted the hint margin margin-left
- Code generation actively ignores default props (if user is using default props, don't show in code snippet)
- Whatever issue I was rambling about in standup this morning (too many props to manage) has been resolved.

huayunh commented:
I just noticed that the ripple is still on even with the `ripple` knob unchecked

https://user-images.githubusercontent.com/8997218/182656913-896bdc15-7376-4371-82ac-a088f6f9657e.mov

emclaug2 commented:
> I just noticed that the ripple is still on even with the `ripple` knob unchecked

I've made an issue for this one, it's a bug on the DrawerNavItem & not the playground: https://github.com/brightlayer-ui/angular-component-library/issues/468

huayunh reviewed (approved):
You have my designer thumbs up! (Didn't check the code)

daileytj commented on src/docs/pages/component-docs/shared/scaffold/scaffold.component.scss:
```diff
@@ -11,36 +11,44 @@
         width: 100%;
         display: flex;
         justify-content: center;
+        .mat-tab-label {
+            // height: 50px;
+        }
```
What's this doing... or not doing?

emclaug2 replied:
There is still some style cleanup required as part of the overall scaffold - making the bottom Divider of the first NavItem align with the divider under the mat-tab-group was intensional.  I had to tweak the height of the tab group for this stuff to align, but it's still not perfect on mobile devices.
